cob_description/urdf/drive_wheel/drive_wheel.urdf.xacro

```diff
@@ -9,17 +9,30 @@
   <xacro:property name="caster_mass" value="5.9" />
   <xacro:property name="wheel_mass" value="0.44036" />
 
-  <xacro:macro name="cob_wheel" params="parent suffix reflect drive_vel:=^">
-
-    <joint name="${parent}_${suffix}_wheel_joint" type="continuous">
-      <origin xyz="0 ${reflect*caster_wheel_offset_y} 0" rpy="0 0 0" />
-      <parent link="${parent}_rotation_link"/>
-      <child link="${parent}_${suffix}_wheel_link"/>
-      <axis xyz="0 1 0" />
-      <safety_controller  k_velocity="10" />
-      <limit effort="100" velocity="${drive_vel}"/>
-      <dynamics damping="0.0" friction="0.0" />
-    </joint>
+  <xacro:macro name="cob_wheel" params="parent suffix reflect drive_vel:=^ passive:=false">
+
+    <xacro:if value="${passive}">
+      <joint name="${parent}_${suffix}_wheel_joint" type="continuous">
+        <origin xyz="0 ${reflect*caster_wheel_offset_y} 0" rpy="0 0 0" />
+        <parent link="${parent}_rotation_link"/>
+        <child link="${parent}_${suffix}_wheel_link"/>
+        <axis xyz="0 1 0" />
+        <safety_controller  k_velocity="10" />
+        <limit effort="0.0" velocity="${drive_vel}"/>
```

Comment:
still needs to be a `continuous` joint in order to allow rotation of wheel - with `effort` set to `0.0` there is no friction/resistance at all thus achieving "passive" wheel behavior....otherwise the front wheels produce to much friction in gazebo for the actuated back wheel to make the robot move...